packages/mermaid/src/diagrams/class/classTypes.ts

```diff
@@ -14,6 +16,95 @@ export interface ClassNode {
   tooltip?: string;
 }
 
+export class ClassMember {
+  id!: string;
+  cssStyle!: string;
+  memberType!: string;
+  visibility!: string;
```

Comment:
Again, is it worth limiting the TypeScript type to just one of these five values?

```suggestion
  visibility!: '#' | '+' | '~' | '-' | '';
```

Reply:
added